### DISCUSSION TITLE
Dual ToneX amp working

### DISCUSSION THREAD

**Discussion by alros100** [Builty/TonexOneController#41] [Show and tell]
Hi Folks

Thought I would share this. Dual Amps !Thanks Builty for making this possible.

I doesn't  appear that the 169 build's midi out functions one its own as a midi through.  so if you want to use two ToneX-One builds you will need to send midi separately to two 169 builds. The big ToneX has midi out/through. So I have midi out from FM9 going to big ToneX then out going to Built's 169 build

If anyone wants the stl files for the 3D printed box let me know. I can share it here or send it to Builty to share.

Al

![image](https://github.com/user-attachments/assets/dbfde52d-0413-4baa-b425-79776b2609bb)

**Comment by Builty** [comment]
Awesome, nicely done.
Yeah I'm not doing Midi out or thru at the moment. I can look at it though
